port/unix/omrsysinfo.c

```diff
@@ -451,6 +451,8 @@ omrsysinfo_get_CPU_architecture(struct OMRPortLibrary *portLibrary)
 	return OMRPORT_ARCH_HAMMER;
 #elif defined(J9ARM)
 	return OMRPORT_ARCH_ARM;
+#elif defined(J9AARCH64)
```

Comment:
I think we have to add this define still.  `J9ARM` is defined in `cmake/modules/platform/arch/arm.cmake:23` and `cmake/modules/platform/arch/arm.cmake:192`

Reply:
Oh, we're not defining AARCH64 either anywhere. I guess this is going in a separate change?

Reply:
this is done in PR  #2526. 
We made the split between port vs build/make relevant stuff. but they pretty much depend on each other to build for aarch64